frontend/src/v4/helpers/comments.ts

```diff
@@ -236,32 +238,32 @@ const convertActionCommentToText = (comment: IComment) => {
 
 			case 'issue_referenced':
 				comment.action.propertyText = 'Referenced';
-				text = 'Issue referenced in #' + comment.action.to + ' by ' + comment.owner;
+				text = 'Issue referenced in #' + comment.action.to + ' by ' + author;
 				break;
 
 			case 'name':
 				comment.action.propertyText = 'Title';
-				text = comment.action.propertyText + ' updated by ' + comment.owner;
+				text = comment.action.propertyText + ' updated by ' + author;
 				break;
 			case 'sequence_start':
 				comment.action.propertyText = 'Sequence start';
-				text = comment.action.propertyText + ' updated by ' + comment.owner;
+				text = comment.action.propertyText + ' updated by ' + author;
 				break;
 			case 'sequence_end':
 				comment.action.propertyText = 'Sequence end';
-				text = comment.action.propertyText + ' updated by ' + comment.owner;
+				text = comment.action.propertyText + ' updated by ' + author;
 				break;
 		}
 	}
 
 	if (0 === text.length) {
 		if (!comment.action.from || !comment.action.to) {
-			text = comment.action.propertyText + ' updated by ' + comment.owner;
+			text = comment.action.propertyText + ' updated by ' + author;
```

Comment:
-  When you reply a comment the mention uses the username:

![reply](https://github.com/user-attachments/assets/1b8efe97-5f6c-4fd3-9fec-d3b674f8a9f0)

Reply:
@The-Daniel  two things:
 -  [ ] Full name is preferred instead of just last name
 -  [ ] Hover on the name is not showing the user card like with the username
 
<img width="358" alt="image" src="https://github.com/user-attachments/assets/d416debc-ec00-442d-adba-8d164048d43d" />